src/gui/accountstate.h

```diff
@@ -217,6 +217,7 @@ private Q_SLOTS:
 
     void slotCheckConnection();
     void slotPushNotificationsReady();
+    void slotUserOnlineStatusChanged();
```

Comment:
I would prefer something like `slotServerUserStatusChanged` to make it easy to get that is coming from the server user status
online could be confused with the state of the user status